Sources/MessagingInApp/MessagingInAppImplementation.swift

```diff
@@ -99,13 +99,10 @@ extension MessagingInAppImplementation: GistDelegate {
         logger.debug("in-app action made. \(action), \(message.describeForLogs)")
 
         // a close action does not count as a clicked action.
```

Comment:
```suggestion
```

We want to remove this comment since the comment was explaining the old `guard` statement.